api/specs/web-server/_licensed_items_purchases.py

```diff
@@ -0,0 +1,55 @@
+""" Helper script to generate OAS automatically
+"""
+
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+
+from typing import Annotated
+
+from _common import as_query
+from fastapi import APIRouter, Depends
+from models_library.api_schemas_webserver.licensed_items_purchases import (
+    LicensedItemPurchaseGet,
+)
+from models_library.generics import Envelope
+from models_library.rest_error import EnvelopedError
+from simcore_service_webserver._meta import API_VTAG
+from simcore_service_webserver.licenses._exceptions_handlers import _TO_HTTP_ERROR_MAP
+from simcore_service_webserver.licenses._models import (
+    LicensedItemsPurchasesListQueryParams,
+    LicensedItemsPurchasesPathParams,
+)
+from simcore_service_webserver.wallets._handlers import WalletsPathParams
+
+router = APIRouter(
+    prefix=f"/{API_VTAG}",
+    tags=[
+        "licenses",
+    ],
+    responses={
+        i.status_code: {"model": EnvelopedError} for i in _TO_HTTP_ERROR_MAP.values()
+    },
+)
+
+
+@router.get(
```

Comment:
TIP: There is a wallets tag that you can add here as well. Check how to add extra tags in the other

Reply:
added